### ISSUE TITLE
[Feature] Scan in BepInEx/config/moresuits folder

### ISSUE THREAD

**Issue by Regnareb** [x753/Lethal-Company-More-Suits#47]
Hi,
Would it be possible to also scan in the BepInEx/config/moresuits folder? 
This way we could share custom suits with only a code on Thunderstore without having to upload a mod.

For privacy reasons I don't want to upload a mod for private jokes, and it would be easier to share to friends

**Comment by x753** [comment]
I don't believe the mod manager syncs files like images, and I don't plan on adding a way to make it download suits from a URL while in-game. You're probably best off sharing your mod(s) manually in a zip file.

**Comment by Regnareb** [comment]
I tried it in the /config folder, any files in the config folder is uploaded/downloaded (even .exe)
Sharing a zip is impossible, having mods through Thunderstore is already a technical hassle for people. Even more impossible when you have to do it each time.

**Comment by x753** [comment]
You're correct, I'll look into this at some point.
